### PR TITLE
C26917: Misuse of ´ to create codeboxes at line 179

### PR DESCRIPTION
**Pull request by v-maudel**
Hello, @mairaw ,
Localization team has reported source content issue that causes localized version to have broken/different format compared to en-us version.  
"this bug is affected by a source issue by the ´ used to create codeboxes"  

Please review and merge the proposed file change to fix to target versions. If you make related fix in another PR  then share your PR number so we can confirm and close this PR.

Many thanks in advance.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/framework/wcf/extending/configuration-and-metadata-support.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ if (transportBindingElement is UdpTransportBindingElement)
  The custom binding element can export policy assertions in the WSDL binding for a service endpoint to express the capabilities of that binding element. The following example code is taken from the [Transport: UDP](../../../../docs/framework/wcf/samples/transport-udp.md) sample.  
   
 #### Policy Export  
- The `UdpTransportBindingElement` type implements``<xref:System.ServiceModel.Description.IPolicyExportExtension?displayProperty=nameWithType> to add support for exporting policy. As a result, <xref:System.ServiceModel.Description.MetadataExporter?displayProperty=nameWithType> includes `UdpTransportBindingElement` in the generation of policy for any binding that includes it.  
+ The `UdpTransportBindingElement` type implements `<xref:System.ServiceModel.Description.IPolicyExportExtension?displayProperty=nameWithType>` to add support for exporting policy. As a result, `<xref:System.ServiceModel.Description.MetadataExporter?displayProperty=nameWithType>` includes `UdpTransportBindingElement` in the generation of policy for any binding that includes it.
```
Comment (BillWagner) [inline docs/framework/wcf/extending/configuration-and-metadata-support.md]: The correct fix would be to remove the code fence character on all the xrefs. The xref will style the element correctly.

**Review by BillWagner — CHANGES_REQUESTED** [review]
@v-maudel The original text was a mistake. However, the fix you've proposed has other issues. Once you've addressed that comment, I'll :shipit:

**Review by BillWagner — APPROVED** [review]

**Comment by v-maudel** [comment]
Thanks so much for the notice @BillWagner  ,
I've already updated the file as suggested and will let the production team to know this as well so we don't propose the same change in the future.

**Comment by v-maudel** [comment]
#sign-off

**Comment by v-maudel** [comment]
Hi @BillWagner , It seems I can't merge it automatically, could you please do it for me or maybe indicate if these changes will be made in another PR?

Thanks.

**Comment by BillWagner** [comment]
@v-maudel I'll merge. I was just waiting for build to finish.

**Comment by v-maudel** [comment]
Thanks  @BillWagner . 
Didn't want to rush you.  :)

**Pull request merged**